File: Modul-2/Soal-5/PRAK205-2410817110016-MuhammadAzriRahman.py
total_input = []
print("Masukan Nilai A dan B: ")
while len(total_input) < 2:
     nilai = input("")
     total_input.extend(nilai.split())

a, b = total_input
a, b = map(int, (a, b))

alas = int((b*b-a*a) ** 0.5)
tinggi = int(a)
keliling = int(a+b+alas)
luas = int((alas*tinggi) / 2)

print(f"Alas = {alas} cm\nTinggi = {tinggi} cm\nKeliling = {keliling} cm\nLuas = {luas} cm^2")

File: Modul-2/Soal-5/PRAK205-2410817110016-MuhammadAzriRahman.c
#include <stdio.h>
#include <math.h>

void main() {
    int a, b;
    int alas, tinggi, keliling, luas;

    printf("Masukan Nilai A dan B: ");
    scanf("%d %d", &a, &b);

    alas = sqrt(b*b-a*a);
    tinggi = a;
    keliling = a+b+alas;
    luas = (alas * tinggi) /2;

    printf("Alas = %d cm\nTinggi = %d cm\nKeliling = %d cm\nLuas = %d cm^2\n",alas, tinggi, keliling, luas);

}